lib/components/molecules/Accordion/Accordion.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+/**
+ *
+ * Accordion
+ *
+ */
+import React, { useState } from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './Accordion.style';
+import type { AccordionHeaderProps, AccordionPanelProps, AccordionProps } from './types';
+
+const AccordionHeader = ({
+  children,
+  className,
+  ariaLabel,
+  accordionPanelId,
+}: AccordionHeaderProps): Node => {
+  const [isExpanded, toggleExpanded] = useState(false);
+
+  const handleClick = () => toggleExpanded(!isExpanded);
+
+  const handleOnKeyPress = event => event.keyCode === 13 && toggleExpanded(!isExpanded);
+
+  return (
+    <summary
+      className={className}
+      role="button"
+      aria-expanded={isExpanded}
+      aria-label={ariaLabel}
+      aria-labelledby={accordionPanelId}
+      onClick={handleClick}
+      onKeyPress={handleOnKeyPress}
+      tabIndex="0"
+    >
+      {children}
+    </summary>
+  );
+};
+
+AccordionHeader.defaultProps = {
+  className: 'accordion-header',
+  ariaLabel: 'aria-label',
+};
+
+/**
+ * AccordionPanel
+ *
+ */
+const AccordionPanel = ({ className, children, id }: AccordionPanelProps): Node => {
+  const [isHidden, toggleHidden] = useState(true);
+  const isIE = false || !!document.documentMode;
+  const [setActive, setActiveState] = useState('');
+  const toggleAccordion = () => {
+    if (isIE) setActiveState(setActive === '' ? 'active' : '');
+    toggleHidden(!isHidden);
+  };
+  const childrenWithHeaderProps = React.Children.map(children, (accordionheader, i) => {
+    if (accordionheader.type.displayName === AccordionHeader.displayName)
+      return React.cloneElement(accordionheader, { accordionPanelId: id });
+
+    if (i === 1)
```

Comment:
I think this condition should not be here since it limits you to only pick the second child. Please achieve this with renderProp